libsolidity/formal/BMC.cpp

```diff
@@ -158,6 +158,7 @@ bool BMC::visit(FunctionDefinition const& _function)
 	{
 		reset();
 		initFunction(_function);
+		m_context.addAssertion(m_context.state().txConstraints(_function));
```

Comment:
Not sure if we should override `initFunction` for BMC, or this is sufficient.

Reply:
I think this is fine